.ci/codenarc.sh

```diff
@@ -1,6 +1,5 @@
 #!/bin/bash
 # Attention, there is no "-x" to avoid problems on Travis
-set -e
```

Comment:
No need to exit if `groovy ./.ci/codenarc.groovy` fails, we still need to echo the reason for failure, for example, if groovy is not installed or `JAVA_HOME` is not set.

Reply:
This is required to fail fast and do not continue execution I'd something existing with error code.

Please share more details and log on this update

Reply:
Actually `set -e` can stay, error in groovy installation or `JAVA_HOME` would be shown in `before_script`

Reply:
Yes, placing `-e` in Travis level code is damaging it's execution, but in script it is ok